mozilla/bedrock
# Update Reps Council on Leadership page after H1 2022 elections

## One-line summary

This adjusts the Reps Council group on the Leadership page after the latest H1 2022 Reps Council elections. Announcement here: https://discourse.mozilla.org/t/important-council-elections-h1-2022-results-announcement/96411

## Significant changes and points to review

* 3 people removed, 3 new people added
* New images have been run through ImageOptim

## Screenshots

![image](https://user-images.githubusercontent.com/330324/165805683-d519cfd1-be1e-4ad9-8180-d6fd89cf6f84.png)

To test this work:
* Go to `/en-US/about/leadership/#reps-council`
* Make sure all images show up and links work correctly
